Patches/Alpha Animals/ThingDefs_Races/AlphaAnimals_CE_Patch_Race_Erin.xml

```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+	<Operation Class="PatchOperationFindMod">
+		<mods>
+			<li>Alpha Animals</li>
+		</mods>
+		<match Class="PatchOperationSequence">
+			<operations>
+				<li Class="PatchOperationAddModExtension">
+					<xpath>/Defs/ThingDef[defName="AA_Erin"]</xpath>
+					<value>
+						<li Class="CombatExtended.RacePropertiesExtensionCE">
+							<bodyShape>Quadruped</bodyShape>
+						</li>
+					</value>
+				</li>
+				<!-- Don't know about balancing -->
+				<li Class="PatchOperationAdd">
+					<xpath>/Defs/ThingDef[defName="AA_Erin"]/statBases</xpath>
+					<value>
+						<ArmorRating_Blunt>2</ArmorRating_Blunt>
+						<ArmorRating_Sharp>0.10</ArmorRating_Sharp>
+						<MeleeDodgeChance>0.40</MeleeDodgeChance>
+						<MeleeCritChance>0.20</MeleeCritChance>
+						<MeleeParryChance>0.10</MeleeParryChance>
+					</value>
+				</li>
+				<li Class="PatchOperationReplace">
+					<xpath>/Defs/ThingDef[defName="AA_Erin"]/tools</xpath>
+					<value>
+						<tools>
+							<li Class="CombatExtended.ToolCE">
+								<label>left claw</label>
+								<capacities>
+									<li>Scratch</li>
```

Comment:
In vanilla, an erin in melee is essentially a slower, harder hitting warg. Use this as an yet another starting point.

Reply:
Done(?)